src/OrchardCore.Modules/OrchardCore.ContentTypes/Services/ContentDefinitionService.cs

```diff
@@ -479,18 +478,19 @@ private static string VersionName(string name)
             int version;
             var nameParts = name.Split('-', StringSplitOptions.RemoveEmptyEntries);
 
-            if (nameParts.Length > 1 && int.TryParse(nameParts.Last(), out version))
+            if (nameParts.Length > 1 && Int32.TryParse(nameParts.Last(), out version))
             {
                 version = version > 0 ? ++version : 2;
-                //this could unintentionally chomp something that looks like a version
-                name = string.Join("-", nameParts.Take(nameParts.Length - 1));
+
+                // This could unintentionally chomp something that looks like a version.
+                name = String.Join("-", nameParts.Take(nameParts.Length - 1));
             }
             else
             {
                 version = 2;
             }
 
-            return string.Format("{0}-{1}", name, version);
+            return String.Format("{0}-{1}", name, version);
```

Comment:
I thought about it, as we use elsewhere, don't remember what is better in term of perf, so here also because the analyzer doesn't suggest anything, maybe another PR on string interpolation if worth to do.